<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeDialog</class>
 <widget class="QDialog" name="CustomizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Font:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color For:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="colorForComboBox">
       <item>
        <property name="text">
         <string>Background</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Foreground</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Color Picker:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="colorPickerButton">
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="attributeListWidget">
     <item>
      <property name="text">
       <string>General Appearance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Table Grid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Labels</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="previewTextLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Preview Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="titleBarCheckBox">
         <property name="text">
          <string>Show Title Bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="toolBarCheckBox">
         <property name="text">
          <string>Show Tool Bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="labelsOnCheckBox">
         <property name="text">
          <string>Labels On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="gridCheckBox">
         <property name="text">
          <string>Show Grid</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Open With</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="defaultRadio">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sizePositionRadio">
          <property name="text">
           <string>Last Size/Pos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="lastSavedRadio">
          <property name="text">
           <string>Last Saved Workspace</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
